keyVaultClientSideToCustomerProvidedServerSide/setup/setup.py

```diff
@@ -0,0 +1,66 @@
+from azure.storage.blob import BlobServiceClient
+from azure.keyvault.keys import KeyClient
+from azure.identity import ClientSecretCredential
+from azure.keyvault.keys.crypto import CryptographyClient, EncryptionAlgorithm
+from keyVaultClientSideToCustomerProvidedServerSide.setup import config as cfg
+
+
+def make_key(k_client):
+    # creates an rsa key by name and saves key to keyvault
+    rsa_key = k_client.create_rsa_key(cfg.keyname, size=2048)  # replace with your keyname
+    k_name = rsa_key.name
+    # return keyvault keyname
+    return k_name
+
+
+def get_key(k_client, k_name):
+    # retrieves key from keyvault by name provided in make_key()
+    keyvault_key = k_client.get_key(k_name)
+    # return keyvault key content
+    return keyvault_key
+
+
+def encrypt_blob(keyvault_key, my_credential):
+    # encrypts blob in bytes using keyvault key from get_key() and credentials from environmental variables
+    message = b"customer provided test"
```

Comment:
is there a constants page in python we can put a constant magic string in?

Reply:
usually we declare constants at the top of the file